sample-input/benchmarks/LRA/geometry.py

```diff
@@ -0,0 +1,201 @@
+import openmoc
+
+###############################################################################
+#                          Main Simulation Parameters
+###############################################################################
+
+options = openmoc.options.Options()
+
+num_threads = options.getNumThreads()
+track_spacing = options.getTrackSpacing()
+num_azim = options.getNumAzimAngles()
+tolerance = options.getTolerance()
+max_iters = options.getMaxIterations()
+
+openmoc.log.set_log_level('NORMAL')
+
+openmoc.log.py_printf('TITLE', 'Simulating the LRA Benchmark Problem...')
```

Comment:
The title gets printed twice since it is also in the main input file so I'd take this out.